docs/contributing/index.md

```diff
@@ -6,7 +6,7 @@ with the community and maintainers via
 [a GitHub Discussion](https://github.com/nsidc/earthaccess/discussions),
 or [any other method](our-meet-ups.md).
 
-Please note that we have a [code of conduct](/CODE_OF_CONDUCT.md). Please follow it in all of your interactions with the project.
+Please note that we have a [code of conduct](./code-of-conduct.md). Please follow it in all of your interactions with the project.
```

Comment:
wonder why strict mode didn't catch this?

Reply:
@mfisher87 Absolute links aren't officially supported on MkDocs because they can be unpredictable and are not validated because they are effectively external links.
https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages

For example, if you host multiple things on a domain, this is a way to link them without knowing the explicit domain name, e.g.:

* root: <pacakge.com>
* MkDocs site URL: <package.com/docs/>
* download artifacts: <example.com/downloads/>

So `/downloads/package-linux.tar.gz` would be a valid link for a tarball you might want to document on the site.


This is probably a good policy as it's really hard to say what `/` is intended to be --  here, the absolute link was intended to be back to the *repository root*, not to the docs dir or the domain root.